lib/bootloader_setup.pm

```diff
@@ -337,6 +338,13 @@ sub specific_bootmenu_params {
         }
     }
 
+    # For leap 42.3 we don't have addon_products screen
+    if (!addon_products_is_applicable()) {
+        my $addon_url = get_var("ADDONURL");
+        $addon_url =~ s/\+/,/g;
```

Comment:
+ is actually a rather unsual seperator - it might have considered useful once :)